Delphi/StalinSortByDelphi.dpr

```diff
@@ -0,0 +1,33 @@
+(*
+ * Stalin Sort by Delphi 10.3.2
+ * If you want to compile this code, please download Delphi 10.3.2
+ *   https://www.embarcadero.com/jp/products/delphi/starter
+ *)
+
+program StalinSortByDelphi;
+
+{$APPTYPE CONSOLE}
+
+function StalinSort(const iTarget: TArray<Integer>): TArray<Integer>;
+begin
+  Result := [];
+
+  var H := High(iTarget);
+  if H < 0 then
+    Exit;
+
+  Result := [iTarget[0]];
+
+  for var i := 1 to H do
+    if Result[High(Result)] < iTarget[i] then

```

Comment:
I don't know delphi, but this probably should be <=
what is the result of a list with repeated numbers like `1 2 4 4 3 5`?

Reply:
I'm wrong as you pointed out